core/src/banking_stage.rs

```diff
@@ -697,45 +699,27 @@ impl BankingStage {
                     .increment_consume_buffered_packets_us(consume_buffered_packets_time.as_us());
             }
             BufferedPacketsDecision::Forward => {
-                let (_, forward_time) = measure!(
-                    Forwarder::handle_forwarding(
-                        cluster_info,
-                        unprocessed_transaction_storage,
-                        poh_recorder,
-                        socket,
-                        false,
-                        data_budget,
-                        slot_metrics_tracker,
-                        banking_stage_stats,
-                        connection_cache,
-                        tracer_packet_stats,
-                        bank_forks,
-                    ),
-                    "forward",
-                );
-                slot_metrics_tracker.increment_forward_us(forward_time.as_us());
+                let (_, forward_us) = measure_us!(forwarder.handle_forwarding(
```

Comment:
nit: `let ((), forward_us) = ...`. so that it's more clear that there's no return value and rustc can complain if ever we change the return type of `.handle_forwarding()` in the future.

Reply:
Nice, didn't even know you could do that!
Done: b7f6913d99269dfeb2c2ab2fc6d85cc9f917feec